server/service/osquery_utils/queries.go

```diff
@@ -561,6 +561,27 @@ FROM
 		DirectIngestFunc: directIngestOrbitInfo,
 		Discovery:        discoveryTable("orbit_info"),
 	},
+	"disk_encryption_darwin": {
+		Query:            `SELECT 1 FROM disk_encryption WHERE user_uuid IS NOT "" AND filevault_status = 'on' LIMIT 1;`,
+		Platforms:        []string{"darwin"},
+		DirectIngestFunc: directIngestDiskEncryption,
+		// the "disk_encryption" table doesn't need a Discovery query as it is an official
+		// osquery table on darwin and linux, it is always present.
+	},
+	"disk_encryption_linux": {
+		Query:            `SELECT 1 FROM disk_encryption WHERE encrypted = 1 AND name LIKE '/dev/dm-1';`,
```

Comment:
What is the difference between using `name LIKE '/dev/dm-1'` and `name = '/dev/dm-1'`?

Reply:
Good catch, I didn't notice that, I just used the queries provided in the ticket. @GuillaumeRoss any reason this is using a `LIKE`? Seems like it should either be `=` or it's missing a `%` at the end?

Reply:
Yeah it 100% seems like using LIKE was a mistake - probably from iterating the query from something else, but if we look at dm-1 only let's use an `=`